external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/internal/Timer.java

```diff
@@ -0,0 +1,60 @@
+package org.apache.storm.kafka.spout.internal;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by liurenjie on 22/12/2016.
```

Comment:
You should really just edit the template in your IDE :)

I'm making the same change here https://github.com/apache/storm/pull/1832/files#diff-73306e32282def8d045e0bcd2d33f59b btw.